### PR TITLE
feat: Add consequence button not showing correct text

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: site/pages/disruption-detail/disruption-detail.test.tsx [hunk 1]
```diff
@@ -302,5 +302,31 @@ describe("pages", () => {
 
             unmount();
         });
+        it("should render correctly with inputs and no errors when disruption has no consequences", () => {
```
Comment (alvak1) [inline site/pages/disruption-detail/disruption-detail.test.tsx:305]: Add a snapshot to check the text "Add a consequence" when number of consequences are zero

Comment (alainajaffer) [reply site/pages/disruption-detail/disruption-detail.test.tsx:305]: I believe that is showing in the snapshot

Comment (alainajaffer) [reply site/pages/disruption-detail/disruption-detail.test.tsx:305]: made tests more specific

**Review by alvak1 — APPROVED** [review]
Approve

**Pull request merged**